docs/pages/index/index.page.server.tsx

```diff
@@ -20,15 +20,13 @@ function Page() {
   )
 }
 
-function GlobalNoteRename() {
+function GlobalNote() {
   return (
     <>
       <div style={{ display: 'flex', justifyContent: 'center', textAlign: 'center' }}>
         <blockquote>
           <p>
-            <a href="https://vite-plugin-ssr.com"></a>
-            <code>vite-plugin-ssr</code> has been renamed Vike, see{' '}
-            <a href="https://vite-plugin-ssr.com/vike">migration guide</a>.
+            The <i>V1 design</i> has been released, see <Link text="migration guide" href="/migration/v1-design" />.
```

Comment:
Is this ok?

Reply:
Good idea 💯

Reply:
I'm seeing that there are still [1/3 of people on vite-plugin-ssr](https://www.npmjs.com/package/vite-plugin-ssr). How about we have both? E.g. with `<ul>`.

Reply:
Fixed, I've decided to go for two thinner notes on top of each other:

![Screenshot from 2023-11-04 10-52-31](https://github.com/vikejs/vike/assets/11795312/946dc67f-e1a8-4dad-9d3a-d39efaa6cd72)

Reply:
👍